tests/providers/gcp/gcp_test.py

```diff
@@ -634,8 +695,75 @@ def testExecuteQuery(self, mock_gcl_api):
     self.assertEqual(FAKE_LOG_ENTRIES[0], query_logs[0])
 
 
+class GoogleCloudStorageTest(unittest.TestCase):
+  """Test Google Cloud Storage class."""
+  # pylint: disable=line-too-long
+
+  @typing.no_type_check
+  def testSplitGcsPath(self):
+    """Tests that GCS path split is correctly done."""
+    bucket, object_uri = FAKE_GCS.SplitGcsPath(FAKE_GCS_PATH)
+    self.assertEqual(object_uri, FAKE_GCS_URI)
+    self.assertEqual(bucket, FAKE_GCS_BUCKET)
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testGetOperationObject(self, mock_gcs_api):
+    """Test GCS object Get operation."""
+    api_get_object = mock_gcs_api.return_value.objects.return_value.get
+    api_get_object.return_value.execute.return_value = MOCK_GCS_OPERATION_OBJECT_GET
+    get_results = FAKE_GCS.GetOperationObject('gs://Fake_Path')
+    self.assertEqual(get_results, MOCK_GCS_OPERATION_OBJECT_GET)
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testGetMD5Object(self, mock_gcs_api):
+    """Test GCS object MD5 hash."""
+    api_get_object = mock_gcs_api.return_value.objects.return_value.get
+    api_get_object.return_value.execute.return_value = MOCK_GCS_OPERATION_OBJECT_GET
+    md5_base64 = FAKE_GCS.GetMD5Object('gs://Fake_Path')
+    self.assertEqual(md5_base64, MOCK_GCS_OPERATION_OBJECT_GET['md5Hash'])
+    md5_hex = FAKE_GCS.GetMD5Object('gs://Fake_Path', in_hex=True)
+    self.assertEqual(md5_hex, base64.b64decode(MOCK_GCS_OPERATION_OBJECT_GET['md5Hash']).hex())
```

Comment:
When testing, please always be as explicit as possible in your comparisons and use literals whenever possible, e.g.

```python
self.assertEqual(md5_hex, 'abcdef....')
```

This makes the test much more clear for anyone wondering what `md5_hex` is expected to contain, and also prevents bugs in your test (in this case, it seems that `MOCK_GCS_OPERATION_OBJECT_GET['md5Hash']`is not a base64 string (`fakehash`), so I wonder how `b64decode` will work on that)

(Bringing this to @binglot's attention as we talked about testing before)

Reply:
Done. Actually b64decode works, it doesn't error and that is the relevant things for this test.